sos/plugins/sssd.py

```diff
@@ -29,6 +29,7 @@ def setup(self):
         self.add_copy_spec([
             "/etc/sssd/sssd.conf",
             "/var/log/sssd/*",
+            "/var/lib/sss/pubconf/krb5.include.d/*",
```

Comment:
Any possible secrets in this location? (I'm assuming not, but could use a sign-off as it's an auth sensitive component).